qiskit/providers/ibmq/api_v2/clients/account.py

```diff
@@ -154,7 +154,12 @@ def job_download_qobj_object_storage(self, job_id):
         download_url = job_api.download_url()['url']
 
         # Download the result from object storage.
-        return job_api.get_object_storage(download_url)
+        object_storage = job_api.get_object_storage(download_url)
```

Comment:
Minor, but can you find a name for this variable that better reflects its content? The function `job_api.get_object_storage` actually means "get (the job) via object storage" (and returns a dict that we can use for reconstructing a job) - off the top of my head, either going with a generic name (`response`) or a more specific one (`job_something`) sounds good.

Reply:
I can think of job_response/response_job, job_result, resulting_job 😄

Reply:
Renamed to `qobj_payload`

Reply:
After the move to the right method, can you revise the variable name again?

Reply:
Idk Github is being weird again. If you go to `Files Changed`, you'll see:
```
# Download the result from object storage.
qobj_payload = job_api.get_object_storage(download_url)

# Notify the API via the callback
_ = job_api.callback_download()

return qobj_payload
```